crates/sozo/ops/src/migration/mod.rs

```diff
@@ -269,7 +275,17 @@ where
         )
         .await?;
 
-        match auto_authorize(ws, &world, &txn_config, &default_namespace, &grant, &revoke).await {
+        match auto_authorize(
+            ws,
+            &world,
+            &txn_config,
```

Comment:
this create an awkward result, because when u run the migration, only this transaction will get debug'd. we only care about contract verification for `sozo migrate` so we shouldn't debug this tx, right?.

<img width="1258" alt="Screenshot 2024-09-05 at 2 38 54 PM" src="https://github.com/user-attachments/assets/8f0a8d23-1d07-4065-8bf6-0b80f36360f6">

i think for quick fix, what we can do here is to create another copy of `TxnConfig` inside of `auto_authorize` and disable the walnut flag before passing to `grant_writer` and `revoke_writer`.

https://github.com/dojoengine/dojo/blob/37887c976ab2cbb78f0d10ae433b0c742d780464/crates/sozo/ops/src/migration/auto_auth.rs#L9-L27